migrator/migrations/frozenschema/v74/permission_sets.go

```diff
@@ -0,0 +1,36 @@
+// Code generated by pg-bindings generator. DO NOT EDIT.
```

Comment:
So I have a fundamental question with this one, Probably for outside the scope of the PR.  The schema did not change from v73.  So why do we need a copy of it in v74?  I think we only need a frozen schema IF the schema changes for the release.  For migrations that do not change the schema, the previous migrations would have been responsible to get the database into the proper schema state and thus a schema that does not change the schema should not try to apply it.  But like I said, just dropping a thought that we should probably take outside the PR.

Reply:
Yes, a migration that does not change schema do not need to apply it but it may still need to use it. This is to make migrations independent of each other for the clarity of the codes. In most cases, we do not have the comprehensive knowledge which schema is changed and which is not. One thing we can do but not yet is to have a version per table for each schema change.